controller/konnect/watch_konnectcontrolplane.go

```diff
@@ -26,7 +29,22 @@ func KonnectGatewayControlPlaneReconciliationWatchOptions(
 ) []func(*ctrl.Builder) *ctrl.Builder {
 	return []func(*ctrl.Builder) *ctrl.Builder{
 		func(b *ctrl.Builder) *ctrl.Builder {
-			return b.For(&konnectv1alpha1.KonnectGatewayControlPlane{})
+			return b.For(
+				&konnectv1alpha1.KonnectGatewayControlPlane{},
+				builder.WithPredicates(
+					predicate.Or(
+						predicate.GenerationChangedPredicate{},
```

Comment:
So this means only enqueue `update`s of controlplanes when spec changed? I think comments should be added here.

Reply:
Removed these.